src/System.Reflection.TypeExtensions/pkg/System.Reflection.TypeExtensions.pkgproj

```diff
@@ -10,10 +10,7 @@
     <HarvestIncludePaths Include="lib/netcore50;runtimes/aot/lib/netcore50" />
     <HarvestIncludePaths Include="ref/netstandard1.3" />
     <HarvestIncludePaths Include="ref/netstandard1.5" />
-    <HarvestIncludePaths Include="lib/netstandard1.5">
-      <!-- this assembly used types / members not exposed by netstandard1.5-->
-      <TargetPath>lib/netcoreapp1.0</TargetPath>
-    </HarvestIncludePaths>
+    <HarvestIncludePaths Include="lib/netcoreapp1.0" />
```

Comment:
Is this going to break again once we start harvesting 4.5.0 packages?

Reply:
True, the 4.5.0 packages are missing the dll completely.  We need to pick it up from the 4.4.0 package.  For that we'll need a depproj.

Reply:
Will the fix be specifying manually the HarvestPackageVersion? Or to set a different targetPath to the NS1.5?

Reply:
I tried to use HarvestAdditionalPackages and it almost worked, except we passed all of HarvestIncludePaths to it as well, so rather than do that I just added a target that did nearly the same thing except let me define the exact things I wanted to pull from the previous package.

Reply:
And to simplify further I decided just to hardcode the HarvestVersion to the 4.4.0 package which was correct.  I added a target to remind us to update this once the latest stable package has the fix.